kafka/source/pkg/reconciler/kafkasource.go

```diff
@@ -57,6 +59,8 @@ import (
 	pkgLogging "knative.dev/pkg/logging"
 )
 
+//KEDA API keda_v1alpha1 "github.com/kedacore/keda/pkg/apis/keda/v1alpha1"
```

Comment:
so, with this landing, we would have a hard dependency on KEDA ... 

Is there some body of work for a more generic integration, and perhaps offer APIs via `knative/pkg`, which we than could consume as some layer?  Or Framework for integrations like this ?

Reply:
I have removed it the commented - it was only to show if wanted to use KEDA API we could. If we have KEDA API in knative/pkg we could use that instead of Unstructured that is used now